src/test/resources/features/99d_employment_qa_smoke.feature

```diff
@@ -73,129 +73,128 @@ Feature: Smoke tests for end of year Employment Journeys to DES/IF in QA - Incom
+    When the user selects the yes radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "How much was your total company car benefit?" page
+    When the user selects the amount field and enters a value of 100.00
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get fuel benefit for a company car?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get a company van benefit?" page
+    When the user selects the yes radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "How much was your total company van benefit?" page
+    When the user selects the amount field and enters a value of 300.00
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get fuel benefit for a company van?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get a mileage benefit for using your own car for work?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get accommodation or relocation benefits from this company?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get any travel or entertainment benefits from this company?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get any utility or general service benefits from this company?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get any medical, dental, childcare, education benefits or loans from this company?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did your employer pay any of your Income Tax or incurred costs?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get any reimbursed costs, vouchers or non-cash benefits from your employer?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get any assets from this company?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Check your employment benefits" page
+    And the user clicks the continue button
```

Comment:
the student loans journey has been merged now so it'll need to be added to this flow (after employment benefits)